lib/rules/declaration-block-no-duplicate-properties/index.js

```diff
@@ -38,34 +39,47 @@ const isEqualValueNodes = (nodes1, nodes2) => {
 		const node1 = nodes1[i];
 		const node2 = nodes2[i];
 
+		// Different types indicate different syntaxes.
 		if (typeof node1 === 'undefined' || typeof node2 === 'undefined' || node1.type !== node2.type) {
 			return false;
 		}
 
+		const node1Name = 'name' in node1 ? String(node1.name) : '';
+		const node2Name = 'name' in node2 ? String(node2.name) : '';
+
+		// Custom properties have unknown value syntaxes but are equal for CSS parsers.
+		if (
+			node1.type === 'Identifier' &&
+			node1Name.startsWith('--') &&
+			node2.type === 'Identifier' &&
+			node2Name.startsWith('--')
```

Comment:
[suggest] Can you use `isCustomProperty()` for consistency?

```suggestion
			isCustomProperty(node1Name) &&
			node2.type === 'Identifier' &&
			isCustomProperty(node2Name)
```

Reply:
I should have checked what this function does :)
I assumed it took a declaration as its argument.